webapp/src/views/CrudList.vue

```diff
@@ -371,7 +362,6 @@ export default {
     defaultObject: {},
     // callbacks
     postcreate: () => (undefined),
-    rowclick: () => (undefined),
```

Comment:
This was used to switch to a domain's RR set view

Reply:
yes, but it's currently dead, and besides the click handling was no longer compatible with the new `item.<name>` slots. We should instead be using the `click:row` event provided by v-datatable. I decided to come up with a new way of handling in consistently with the new slots once we have a use case.